src/util.cpp

```diff
@@ -400,6 +400,23 @@ int64_t GetArg(const std::string& strArg, int64_t nDefault)
     return nDefault;
 }
 
+int64_t GetArgWithinLimits(const std::string& strArg, int64_t nDefault, std::vector<std::pair<int64_t, int64_t>> limits)
```

Comment:
I think it might be worth to pass `limits` as a const reference here.

Reply:
Done